data/json/items/gun/22.json

```diff
@@ -21,8 +21,11 @@
     "aim_speed": 3,
     "recoil": 10,
     "durability": 6,
-    "burst": 30,
     "barrel_length": 1,
+    "modes": [
```

Comment:
These form of changes are probably a bad idea for now (and will block your PR) until a consensus is reached as to whether we are going to support both formats concurrently (I would suspect yes). If so such changes are actually obfuscation.